src/main/java/com/endava/config/ConfigConstants/ConfigConstants.java

```diff
@@ -0,0 +1,15 @@
+package com.endava.config.ConfigConstants;
```

Comment:
Shouln't this configs be in the dababase too?

Reply:
Fixed